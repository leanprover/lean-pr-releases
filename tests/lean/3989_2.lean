def foo (ty : Expr) : MetaM Expr :=
  match_expr ty with
  | Nat => sorry
  | BitVec n => sorry

#check Nat
