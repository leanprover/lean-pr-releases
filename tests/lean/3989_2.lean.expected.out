3989_2.lean:6:0: error: expected else-alternative for `match_expr`, i.e., `| _ => ...`
Nat : Type
